scripts/deployToDB.py

```diff
@@ -13,12 +13,16 @@ def get_command_line_arguments():
     parser.add_argument("control_plane_url", nargs="?", help="Control plane URL")
     parser.add_argument("username", nargs="?", help="Control plane admin username")
     parser.add_argument("password", nargs="?", help="Control plane admin password")
+    parser.add_argument(
+        "item_name", nargs="?", help="Specific item name to update.", default=None
```

Comment:
There are some items that have the same name in sources and destinations. We might also need to selector.

Reply:
Added